client/client.js

```diff
@@ -11,6 +11,8 @@ import { clientSyncTitle } from '../plugins/sync-title/client.js';
 import { recordUserEvents } from '../plugins/user-activity-events/client.js';
 import { SlimClient } from './slim.js';
 
+window.ifrauClientLoaded = true;
```

Comment:
I have zero opinion about this naming convention.

Reply:
One thing I did check is that there doesn't appear to be any usage of this anywhere at the moment, so I don't think overwriting any existing behaviour would be a concern.